### DISCUSSION TITLE
0.54.0-ALPHA1 problem

### DISCUSSION THREAD

**Discussion by PH5HP** [rejetto/hfs#689] [General]
Hi Rejetto,

This version produces a blank page here. Only the background is visible.

Kind regards,

Henk - fileserver.ph5hp.nl
![image](https://github.com/user-attachments/assets/b5a15d5d-963e-4bde-bce0-08369b668126)

**Comment by rejetto** [comment]
i cannot reproduce the problem.
maybe one day we can have a closer look together.

**Reply by PH5HP** [reply]
Hi,

I will try that version again. One of my visitors made me aware of this. I let you know if I'll find something.

Kind resgards,
Henk - fileserver.ph5hp.nl - ph5hp.nl

**Comment by PH5HP** [comment]
Hi,

I pressed F12 in Chrome and this is what I get:

![image](https://github.com/user-attachments/assets/c133a4a7-38f7-486b-9a6d-3a830836c935)

Kind regards,
Henk - fileserver.ph5hp.nl - ph5hp.nl

**Comment by rejetto** [comment]
that was useful thanks.
i'll keep you posted.

**Reply by PH5HP** [reply]
Hi,

Any news?

**Comment by maandoz** [comment]
hi
i use win11 64 pro - i have the same problem. this is the second time.
it happened to me also between 052 and 053 (win10 64 pro).
i solved it by re_setting the config.yaml file.
i started from the standard installation, unzip hfs-windows.....
i think that for some reason hfs loses the link with the config.yaml file
it is possible that it edited the config file with hfs closed...
i have not tried with 54 yet
let me know - thanks

**Comment by maandoz** [comment]
hi
i unzip hfs054 in a new folder - i set config.yaml .....
hfs054 works fine
i use opera browser

ps.
i can't update 053 automatically or manually
the config.yaml files hfs053 - hfs054 are the same - the number of lines can change but not their content
eg hfs053 line number 6 http port:1111
     hfs054 line number 8 http port:1111

**Comment by rejetto** [comment]
the problem reported by @PH5HP is caused by an incompatibility introduced by 0.54.0-alpha1 with plugins "rich-folder" and "download-quota".
@maandoz The blank screen can be caused by many things, so your problem may have been different. The red error visible in the screenshot of ph5hp pointed me to the right problem.
To solve it i just published an updated for the both plugins.

**Reply by PH5HP** [reply]
Hi,

Updated to the last version (0.54.0 Alpha 1 and updated after that the 2 plugins mentioned above. Everything is now running Ok. Problem seems to be solved.

Kind regards,
Henk - https://fileserver.ph5hp.nl - https://ph5hp.nl

**Comment by maandoz** [comment]
hi rejetto,
as always you solved it. thanks
